sdk/src/loom2d/Display/DisplayObject.ls

```diff
@@ -180,7 +180,7 @@ package loom2d.display
             parentCached = value;
         }
 
-        /** If depth sorting is enabled on parent, this will be used to establish draw order. */
+        /** If depth sorting is enabled on parent, this will be used to establish draw order. Higher values are drawn closer. */
```

Comment:
1. Added note.
2. Yes. You can iterate over the depth values to get min/max. This is for a Z-sort usage case primarily - if relative ordering is key, then setChildIndex might be a better fit. 
3. There is no difference between int and float internally in Loom.
4. It is a moot point.